ui/src/testing/stubs/stub-utils.tsx

```diff
@@ -0,0 +1 @@
+export const StubImplementationRequired = Error('Cannot fetch in tests.  Please implement stub method.');
```

Comment:
opt nit: Personally I strongly prefer avoiding double space after periods in comments/error messages, but I realize others may have strongly held preferences in the other direction.